api/bertymessenger.proto

```diff
@@ -253,8 +253,8 @@ message ConversationJoin {
 message Account {
   string public_key = 1 [(gogoproto.moretags) = "gorm:\"primaryKey\""];
   string display_name = 2;
-  string link = 3;
-  State state = 4;
+  string link = 3; // FIXME(?): Store RDV seed and generate it instead, provide it in the struct returned via the API
+  State state = 4; // FIXME(?): What is this, how is it used, does it have to be stored in the database
```

Comment:
This is used to signal to the app that the account has been setup and is ready to be used when in Ready state, currently the code that sets it only verifies that the account has a custom name